js/models/base.js

```diff
@@ -515,4 +515,16 @@ export class PageList extends List {
         this.populate();
         return this;
     }
+
+    /**
+     * Perform a server-side search
+     * @param  {String} query The query string to perform the search on.
+     * @return {Object} Return itself allowing to chain methods.
+     */
+    search_server(query) {
```

Comment:
no, I clear paging on data received avoiding going back to first page before new data is present.